File: R/tests/testthat/test_pstRT_ddm.R
context("Test pstRT_ddm")
library(hBayesDM)

test_that("Test pstRT_ddm", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(pstRT_ddm(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})

File: Python/tests/test_pstRT_ddm.py
import pytest

from hbayesdm.models import pstRT_ddm


def test_pstRT_ddm():
    _ = pstRT_ddm(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()
